packages/deployments/contracts/contracts/core/connext/libraries/LibConnextStorage.sol

```diff
@@ -137,10 +136,10 @@ struct AppStorage {
   // 4
   uint32 domain;
   /**
-   * @notice The local nomad token registry.
+   * @notice UpgradeBeacon from which new token proxies will get their implementation
    */
   // 5
-  ITokenRegistry tokenRegistry;
+  address tokenBeacon;
   /**
```

Comment:
where will `tokenBeacon` be initialized? maybe we should deploy separately and init it with diamond init contract

Reply:
it is used as the implementation address used to deploy new bridge token proxies, which are now deployed on `setupAsset`

Reply:
Yes, but i couldn't find where `tokenBeacon` address is initialized.

Reply:
ahh good catch -- ill add it to the deployer